test/testdrive/mzcompose.py

```diff
@@ -96,11 +94,7 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
         validate_postgres_stash=True,
     )
 
-    postgres = Postgres(image="postgres:13.6")
-
-    with c.override(materialized, testdrive, postgres):
-        c.up("postgres")
-        c.wait_for_postgres()
```

Comment:
Note that despite this change we still use PostgreSQL in this composition. We just use the PostgreSQL server embedded in the `materialized` Docker image rather than an explicit service.